prettier/tests/__snapshots__/run_tests.js.snap

```diff
@@ -4103,31 +4129,51 @@ This is a simple dialog rendered by using a macro and a call block.
     use_page_url=True,
     overrideable=True %}
     {% widget_attribute "pinterest"
-      is_json=True %}{"custom_link_format": "", "pinterest_media":
-      "http://cdn2.hubspot.net/hub/158015/305390_10100548508246879_837195_59275782_6882128_n.jpg",
-      "enabled": true, "network": "pinterest", "img_src":
-      "https://static.hubspot.com/final/img/common/icons/social/pinterest-24x24.png"}
-  {% end_widget_attribute %}
+      is_json=True %}
```

Comment:
Do we want the `is_json=True` to always show up on a newline? I would expect it all to be on one line like...

```
{% widget_attribute "pinterest" is_json=True %}
```

Reply:
IIRC this was a request to match the internal CAT Style Guide

Reply:
CAT style guide is more:
```
{% widget_attribute "pinterest"
  is_json=True
%}
```
But we usually would only split up the tag like this when there's a large list of params, for only one like this keeping it one line would be totally acceptable.